### ISSUE TITLE
Where is MTU?

### ISSUE THREAD

**Issue by cornzzy** [naggie/dsnet#94]
How do I set MTU?

**Comment by naggie** [comment]
Good point --- I have started a PR here but still need to figure out a few things. https://github.com/naggie/dsnet/pull/95 -- if you get a sec pls have a go :-)

**Comment by naggie** [comment]
Ok implemented. @frillip is testing later today
